test

```diff
@@ -185,16 +188,12 @@ def test_all(year, event, problem, language, id):
 
   checker = prepare_checker(language, file)
 
-  results = [
-    test(checker, input, expected)
-    for input, expected in test_case
-  ]
+  for input, expected in test_case:
+    yield test(checker, input, expected)
```

Comment:
Fuck. You are right. But check this shit:
![screenshot_2016-09-20_19-42-35](https://cloud.githubusercontent.com/assets/7642878/18691532/8f77feb4-7f6a-11e6-83ee-f1a26764be43.png)

I was be trolled for github. THANKS!!